OracleUnifiedDirectorySM/dockerfiles/buildDockerImage.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at 
+# https://oss.oracle.com/licenses/upl.
+#
+# Description: script to build a Docker image for Oracle Unified Directory Service Manager
+#
+
+usage() {
+cat << EOF
+
+Usage: buildDockerImage.sh -v [version] -d [domain-type] [-s]
+Builds a Docker Image for Oracle Unified Directory Service Manager
+
+Parameters:
+   -h: view usage
+   -v: Release version to build. Required. E.g 12.2.1.4.0
+   -s: skips the MD5 check of packages (DEFAULT)
```

Comment:
The script sets `SKIPMD5=0` by default which means this is not the default.

Reply:
Updated usage doc.